samples/campaign_sample.py

```diff
@@ -67,29 +61,25 @@ def send_the_campaign_right_away():
 
 def api_call_requirements():
     """POST https://api.mailjet.com/v3.1/send"""
+    # fmt: off
     data = {
         "Messages": [
             {
                 "From": {
                     "Email": "pilot@mailjet.com",
-                    "Name": "Mailjet Pilot"
-                },
+                    "Name": "Mailjet Pilot"},
                 "To": [
                     {
                         "Email": "passenger1@mailjet.com",
-                        "Name": "passenger 1"
-                    }
-                ],
+                        "Name": "passenger 1"}],
                 "Subject": "Your email flight plan!",
                 "TextPart": "Dear passenger 1, welcome to Mailjet! May the delivery force be with you!",
                 "HTMLPart": "<h3>Dear passenger 1, welcome to <a "
-                            "href=\"https://www.mailjet.com/\">Mailjet</a>!</h3><br />May the delivery force be with "
-                            "you!",
+                "href=\"https://www.mailjet.com/\">Mailjet</a>!</h3><br />May the delivery force be with "
```

Comment:
This one looks strange, previous version was more readable.

Reply:
Thanks!

Reply:
Fixed